test/test.cc

```diff
@@ -3287,7 +3287,8 @@ TEST(GzipDecompressor, ChunkedDecompression) {
 }
 
 #ifdef _WIN32
-TEST(GzipDecompressor, LargeRandomData) {
+// Disabled due to the out-of-memory problem on GitHub Actions Workflows
+TEST(GzipDecompressor, DISABLED_LargeRandomData) {
```

Comment:
Since this is an issue with GitHub Actions, the test should be disabled there instead of here. You can do so by passing the `--gtest_filter=-GzipDecompressor.LargeRandomData` flag to the test binary or by setting the `GTEST_FILTER` environment variable to the same value